.circleci/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+GLOW_DIR=$PWD
+GLOW_BUILD_DIR=${GLOW_DIR}/build
+cd ${GLOW_BUILD_DIR}
+
+# Run Glow test
+CTEST_PARALLEL_LEVEL=2 ninja test || ( cat Testing/Temporary/LastTest.log && exit 1 )
+if [ "$TEST_NAME" = "DEBUG" ]; then CTEST_PARALLEL_LEVEL=2 ninja test_unopt; fi
```

Comment:
This is not the right check

Reply:
There is no `TEST_NAME`, it's `CIRCLE_JOB`, and also right now you are setting it as "trusty-debug-clang6.0-ubuntu16.04", not "DEBUG".